engine/h2shared/r_bsp.c

```diff
@@ -653,6 +657,7 @@ void R_RenderWorld (void)
 
 	R_RecursiveWorldNode (clmodel->nodes, 15);
 
+#if 0
```

Comment:
The `int i;` needs moving from top into the `if (r_worldpolysbacktofront)` block

Reply:
Indeed, now I see the warning about this.